ts/src/binance.ts

```diff
@@ -5885,7 +5886,8 @@ export default class binance extends Exchange {
         let marginMode = undefined;
         [ marginMode, params ] = this.handleMarginModeAndParams ('createOrder', params);
         const reduceOnly = this.safeBool (params, 'reduceOnly', false);
-        if ((marketType === 'margin') || (marginMode !== undefined) || market['option']) {
```

Comment:
even though I think that we shouldn't be supporting `marginMode` here (because it doesnt have any actual effect and here it's used in a bit vague way) i've left that logic (which is reflected in  `isSpotMargin` var)